gpslogger/src/main/res/values-de/strings.xml

```diff
@@ -364,4 +366,6 @@
     <string name="app_theme_setting">Legt fest, wie das helle oder dunkle App-Thema eingestellt werden soll</string>
     <string name="file_logging_log_time_with_offset_title">Uhrzeit mit Zeitzonenverschiebung protokollieren</string>
     <string name="file_logging_log_time_with_offset_summary">Wenn möglich, geben Sie die ISO8601-Zeit mit Zeitzonen-Offset anstelle von UTC an.</string>
+    <string name="logging_advanced_delete_files">Lösche ausgewählte Dateien</string>
+    <string name="logging_advanced_delete_files_summary">Lang drücken zur Mehrfachauswahl von Dateien</string>
 </resources>
```

Comment:
OK - ignoring these - I see some equivalents in the [Weblate PR](https://github.com/mendhak/gpslogger/pull/948/files)